### DISCUSSION TITLE
Similarity search using embedding

### DISCUSSION THREAD

**Discussion by alex-coch** [milvus-io/milvus#23846] [Q&A and General discussion]
How can I carry out similarity search at DB if I 've got an embedding  (not a text query)? This embedding represents an one paragraph.

**Comment by xiaofan-luan** [comment]
> How can I carry out similarity search at DB if I 've got an embedding (not a text query)? This embedding represents an one paragraph.

Currently what you can do is to get topk result and run another query on it. 
We are working on supporting getVectorByID very soon
But careful this might be reduce the search performance and throughput.
